features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdConfigBean.java

```diff
@@ -0,0 +1,55 @@
+package org.opennms.netmgt.trapd;
+
+import java.util.List;
+
+import org.opennms.netmgt.config.TrapdConfig;
+import org.opennms.netmgt.snmp.SnmpV3User;
+
+public final class TrapdConfigBean implements TrapdConfig {
+
+	private String m_snmpTrapAddress;
+	private int m_snmpTrapPort;
+	private boolean m_newSuspectOnTrap;
+	private List<SnmpV3User> m_snmpV3Users;
+
+	public void setSnmpTrapAddress(String m_snmpTrapAddress) {
+		this.m_snmpTrapAddress = m_snmpTrapAddress;
+	}
+
+	public void setSnmpTrapPort(int m_snmpTrapPort) {
+		this.m_snmpTrapPort = m_snmpTrapPort;
+	}
+
+	public void setM_newSuspectOnTrap(boolean m_newSuspectOnTrap) {
```

Comment:
Remove the M_ from the function name and remove the m_ from all of the function arguments here.